src/Database/Migrations/2020-12-28-223112_create_auth_tables.php

```diff
@@ -22,7 +22,12 @@ public function __construct(?Forge $forge = null)
         parent::__construct($forge);
 
         /** @var Auth $authConfig */
-        $authConfig       = config('Auth');
+        $authConfig = config('Auth');
+
+        if ($authConfig->DBGroup !== null) {
+            $this->DBGroup = $authConfig->DBGroup;
+        }
+
```

Comment:
@arashsaffari If this change is not made, if you set custom `DBGroup`. Migration runs with defaults `DBGroup`. And this is a bug.
Please wait for a member to confirm this.
```diff
-parent::__construct($forge);
-
/** @var Auth $authConfig */
$authConfig       = config('Auth');
$authConfig = config('Auth');

if ($authConfig->DBGroup !== null) {
$this->DBGroup = $authConfig->DBGroup;
}
+parent::__construct($forge);
```

Reply:
Hmm, it seems that using the following is more readable:
 ```php
use Config\Database;


$this->forge = Database::forge($authConfig->DBGroup);
$this->db = $this->forge->getConnection();
```

Reply:
@datamweb is right. `parent::__construct($forge)` should be moved after the assignment of `$this->DBGroup`.

Reply:
@Kenjis, thank you for your confirmation. @arashsaffari  apply the changes.